db/migrate/20170702021929_create_law_offices.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLawOffices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :law_offices do |t|
+      t.string :law_office
+      t.string :contact_information
```

Comment:
That actually sounds like a good idea. Thank you for this advice! I'm going to change this right now